common-app-lib/src/cmr/common_app/api/health.clj

```diff
@@ -53,5 +55,7 @@
   takes a request-context as a parameter to determine if the application and its dependencies are
   working as expected."
   [health-fn]
+  (OPTIONS "/" req common-routes/options-response)
+  (OPTIONS "/health" req common-routes/options-response)
   (GET "/health" {:keys [request-context]}
```

Comment:
Do I need to add a (GET "/" ...)  case?  it won't be able to get the health check result, but it will not trigger CORS error? 
@oubiwann, I'm not sure what it means to have to take care of both "/" and "/health" cases.